index.html

```diff
@@ -2,22 +2,24 @@
 <link rel="stylesheet" href="style/style.css">
 <meta name="viewport" content="width=device-width,initial-scale=1">
 
-**OpenPBR Surface specification**
+**OpenPBR Surface**
```

Comment:
I question this change.
I think it's useful to see in the title that the document is a specification, especially considering that we may have more documents in the future, like an artist guide for example.

Reply:
Note that "specification" and the "version" string could be moved to a sub-title before the author list.

Reply:
How about this? (Added in c81366e84ffbba7dfbdd5ca358b1640e51736657)

![image](https://github.com/AcademySoftwareFoundation/OpenPBR/assets/2774364/6595f510-ffd1-4d64-b1b8-257e067c3886)

Reply:
Yep, I think that works.
(except of course for the additional changes mentioned in the meeting)

Reply:
Should we include the removal of top-level company and author names as part of this changelist, as suggested in today's meeting?

Reply:
OK let me try it.

Reply:
>removal of top-level company 

Do we want to mention ASWF in its place?